Library/Formula/whatmp3.rb

```diff
@@ -0,0 +1,26 @@
+class Whatmp3 < Formula
+  desc "whatmp3 is a small script to create mp3 torrents out of FLACs."
+  homepage "https://github.com/RecursiveForest/whatmp3"
+  url "https://github.com/RecursiveForest/whatmp3/archive/v3.6.tar.gz"
+  sha256 "a1f5ef28e2511aa47f4658f71a8a3afe3dda96e7dd6a1cf9d124daead0fed5fa"
+  head "https://github.com/RecursiveForest/whatmp3.git"
+
+  depends_on "python"
+  depends_on "flac"
+  depends_on "mktorrent" => :recommended
+  depends_on "vorbis-tools" => :optional
+  depends_on "lame" => :optional
+  depends_on "mp3gain" => :optional
+  depends_on "aacgain" => :optional
+  depends_on "vorbisgain" => :optional
+  depends_on "sox" => :optional
+
+  def install
+    bin.install "whatmp3"
+  end
+
+  test do
+    output = shell_output("#{bin}/whatmp3 --version")
+    assert_match /whatmp3 3.6/, output
```

Comment:
We need a test that does something more substantial than e.g. `--version`/`--help`. See the [`flac.rb`](https://github.com/Homebrew/homebrew/blob/31db1e66c71a2378c847070d58f143af4f39b068/Library/Formula/flac.rb#L68) for a test with a dummy FLAC file. `#{bin}/whatmp3 --320 -n something.flac` will convert a file but not create a torrent so we could use that. Please ask if you have any question.